modules/util/source-formatter/src/main/java/com/liferay/source/formatter/CSSSourceProcessor.java

```diff
@@ -155,8 +167,10 @@ protected String sortProperties(String content) {
 
 	private static final String[] _INCLUDES = {"**/*.css", "**/*.scss"};
 
-	private final Pattern _commentPattern =
-		Pattern.compile("/\\* -+(.+)-+ \\*/");
+	private final Pattern _commentFormatPattern = Pattern.compile(
+		"^-* ?(\\S.*?\\S) ?-*$");
```

Comment:
@hhuijser Just an FYI, these changes are now causing some issues with the front end check-source-formatter module, in that we expect all CSS comments (`/* */`) to have a single space after the first `*` and before the last `*`.
But if you look at this guy: https://github.com/brianchandotcom/liferay-portal/pull/43812/files#diff-fcd195013fa1489ba5b4a99c2472fb4aR1, it's stripping out those whitespaces.

Can we switch back to that, so it's consistent everywhere on the frontend?